### ISSUE TITLE
Initial result of data analysis

### ISSUE THREAD

**Issue by swiri021** [OpenKBC/multiple_sclerosis_proj#24]

**Comment by swiri021** [comment]
@kicheolkim  It seems CIS and RR has DiseaseDuration difference significantly (notebook/notebook_archive/Jun09262021/meta_analysis.ipynb), so if we find features that are relevant to early and late markers by using DiseaseDuration, those might be differential expressed genes between RR and CIS. Do you think CIS and RR have actual meaning for the early and late stages of MS? for example, CIS might be really the early stage of MS and RR for the medium stage.

**Comment by kicheolkim** [comment]
@swiri021 I don't think RR can consider as a middle of the stage. As far as I know, in clinics, CIS is just experienced the first time neurodegeneration. If the patient had an attack again, it considers as the RR stage. A patient can have more attacks or have no more attacks after the CIS stage. That's why I think the data is good for disease mechanisms study but may not be good for diagnostic biomarkers.

**Comment by swiri021** [comment]
* Compared AUCs of SVM models between DEG features and activation score features, and activation score model has higher AUC for same CD4 result.
* Notebooks :
[SVM with DEG features](notebook/feature_test_with_DEG.ipynb), [SVM with activation score features](notebook/feature_test_with_act.ipynb)

**Comment by kicheolkim** [comment]
wow... it's very interesting...
Is the state single gene? or gene set?

**Comment by swiri021** [comment]
> wow... it's very interesting... Is the state single gene? or gene set? Is there a list of the genes?

Activation score is calculated by gene-sets(gene signatures), and DEG is one list of DEG as you know... So, here, DEG model is using one gene as one features and Activation Score model is using one pathway score as one feature.

**Comment by swiri021** [comment]
> wow... it's very interesting... Is the state single gene? or gene set? Is there a list of the genes?

I think this may occur because of features numbers, and I downed DEG fold change threshold  to 0.58(1.5 fold) and performance is better than 2 fold threshold. But still, the activation score model has narrower discrepancies of AUC between validation-set and test-set. Anyway, I am going to dig pathway features deeply.

**Comment by kicheolkim** [comment]
So, the activation score is based on pathways? and it's the list from DEG (DESeq2 results)?
Do you have a list of pathway and/or gene sets? I'm curious what pathways/genes are included.

**Comment by swiri021** [comment]
> So, the activation score is based on pathways, and the gene set is from DEG (DESeq2 results)? Do you have a list of pathway and/or gene sets? I'm curious what pathways/genes are included.

Yes, we have a list of pathways, and the activation score was calculated by using MSigDB. Additionally, I will let you know if we have more interesting points here.

**Comment by swiri021** [comment]
@kicheolkim @lacuss I got a weird signal in the data: [Notebook link](https://github.com/OpenKBC/multiple_sclerosis_proj/blob/main/notebook/feature_analysis_with_act.ipynb)
That signal is related to 'Sex' of patients, unfortunately, it is related to RR and CIS category significantly..... maybe another noise?

**Comment by swiri021** [comment]
* CD4, CD8, CD14 all dataset is showing the same PCA pattern (Sex is the highest factor to cluster data in pathways level)

**Comment by lacuss** [comment]
input | name | symbol | alias (first 5) | HGNC
-- | -- | -- | -- | --
8653 | DEAD-box helicase 3 Y-linked | DDX3Y | DBY | HGNC:2699
9086 | eukaryotic translation initiation factor 1A Y-linked | EIF1AY | eIF-4C | HGNC:3252
8284 | lysine demethylase 5D | KDM5D | HY, HYA, JARID1D, SMCY | HGNC:11115
5616 | protein kinase Y-linked (pseudogene) | PRKY | PRKXP3, PRKYP | HGNC:9444
6192 | ribosomal protein S4 Y-linked 1 | RPS4Y1 | RPS4Y, S4 | HGNC:10425
9087 | thymosin beta 4 Y-linked | TMSB4Y | TB4Y | HGNC:11882
246126 | taxilin gamma pseudogene, Y-linked | TXLNGY | CYorf15A, CYorf15B, TXLNG2P | HGNC:18473
8287 | ubiquitin specific peptidase 9 Y-linked | USP9Y | DFFRY, SPGFY2 | HGNC:12633
7404 | ubiquitously transcribed tetratricopeptide repeat containing, Y-linked | UTY | KDM6AL, KDM6C, UTY1 | HGNC:12638
7544 | zinc finger protein Y-linked | ZFY | ZNF911 | HGNC:1

**Comment by swiri021** [comment]
As we expected, all of genes are Y-linked

**Comment by swiri021** [comment]
https://soobarkbar.tistory.com/30
